test/WebJobs.Script.Tests/packages.config

```diff
@@ -6,18 +6,18 @@
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net45" />
   <package id="Microsoft.AspNet.WebHooks.Common" version="1.2.0-rc1" targetFramework="net46" />
   <package id="Microsoft.AspNet.WebHooks.Receivers" version="1.2.0-beta6" targetFramework="net46" />
-  <package id="Microsoft.Azure.ApiHub.Sdk" version="0.3" targetFramework="net46" />
+  <package id="Microsoft.Azure.ApiHub.Sdk" version="0.5" targetFramework="net46" />
   <package id="Microsoft.Azure.DocumentDB" version="1.5.3" targetFramework="net46" />
   <package id="Microsoft.Azure.Mobile.Client" version="2.0.1" targetFramework="net46" />
   <package id="Microsoft.Azure.NotificationHubs" version="1.0.3" targetFramework="net46" />
   <package id="Microsoft.Azure.WebJobs" version="1.1.2-alpha-10289" targetFramework="net46" />
   <package id="Microsoft.Azure.WebJobs.Core" version="1.1.2-alpha-10289" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions" version="1.0.2-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="1.0.0-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="1.0.0-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.EasyTables" version="1.0.0-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.NotificationHubs" version="1.0.0-alpha-10268" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.SendGrid" version="1.0.2-alpha-10268" targetFramework="net46" />
+  <package id="Microsoft.Azure.WebJobs.Extensions" version="1.0.2-alpha-10270" targetFramework="net46" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="1.0.0-alpha-10270" targetFramework="net46" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="1.0.0-alpha-10267" targetFramework="net46" />
```

Comment:
Why are some packages like DocumentDB changing to older versions? That doesn't seem right.